libraries/chain/include/cyberway/genesis/ee_genesis_serializer.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+#include <boost/filesystem/fstream.hpp>
+#include <cyberway/chaindb/common.hpp>
+#include <cyberway/genesis/ee_genesis_container.hpp>
+#include <eosio/chain/abi_serializer.hpp>
+
+namespace cyberway { namespace genesis {
+
+using namespace chaindb;
+namespace bfs = boost::filesystem;
+
+//const fc::microseconds abi_serializer_max_time = fc::seconds(10);
+
+struct ee_genesis_serializer {
```

Comment:
Where do you calling it? For what pull-request with unaccessible code?

Reply:
ee_genesis_serializer used in create-genesis utility. See files in this PR.

Reply:
@s-medvedev But it also should be used in create-genesis-ee.

Reply:
Yes, you are right. But this should be done in another issue.

Reply:
See issue #559